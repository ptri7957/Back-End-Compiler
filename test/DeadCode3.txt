( (main (n)
    (0 (ld r1 x)
       (br r1 1 2) )
    (1 (lc r2 1)
       (add r3 r1 r1)
       (st x r3)
       (ld r4 x)
       (ret r4) )
    (2 (ld r5 n)
       (lc r6 90)
       (sub r7 r5 r5)
       (st n r7)
       (ld r8 n)
       (ret r8) ) ) )