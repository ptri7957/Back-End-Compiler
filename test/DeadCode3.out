( (main (n)
    (0 (ld r1 x)
       (br r1 1 2) )
    (1 (add r3 r1 r1)
       (st x r3)
       (ld r4 x)
       (ret r4) )
    (2 (ld r5 n)
       (sub r7 r5 r5)
       (st n r7)
       (ld r8 n)
       (ret r8) ) ) )